user-interface/src/main/java/life/qbic/datamanager/views/projects/create/ExperimentalInformationLayout.java

```diff
@@ -0,0 +1,268 @@
+    Span experimentInformationDescription = new Span(
+        "The experiment name and sample origin information of the samples");
+    experimentInformationTitle.addClassName("title");
+    add(experimentInformationTitle,
+        experimentInformationDescription);
+  }
+
+  private void initNameField() {
+    TextField experimentNameField = new TextField("Experiment Name");
+    experimentNameField.addClassName("experiment-name-field");
+    experimentNameField.setPlaceholder("Please enter a name for your experiment");
+    experimentNameField.setRequired(true);
+    experimentalInformationBinder.forField(experimentNameField)
+        .withValidator(it -> !it.isBlank(), "Please provide a name for the experiment")
+        .bind(ExperimentalInformation::getExperimentName,
+            ExperimentalInformation::setExperimentName);
+    add(experimentNameField);
+  }
+
+  private void initOntologyComboboxes() {
+    MultiSelectComboBox<Species> speciesBox = new MultiSelectComboBox<>("Species");
+    initComboBoxWithDatasource(speciesBox, List.of(Ontology.NCBI_TAXONOMY),
+        term -> new Species(term.getLabel()));
+    speciesBox.setItemLabelGenerator(Species::label);
+    speciesBox.setPlaceholder("Please select one or more species for your samples");
+    experimentalInformationBinder.forField(speciesBox)
+        .asRequired("Please select at least one species")
+        .bind(experimentalInformation -> new HashSet<>(experimentalInformation.getSpecies()),
+            ExperimentalInformation::setSpecies);
+    MultiSelectComboBox<Specimen> specimenBox = new MultiSelectComboBox<>("Specimen");
+    initComboBoxWithDatasource(specimenBox,
+        Arrays.asList(Ontology.PLANT_ONTOLOGY, Ontology.BRENDA_TISSUE_ONTOLOGY),
+        term -> new Specimen(term.getLabel()));
+    specimenBox.setItemLabelGenerator(Specimen::label);
+    specimenBox.setPlaceholder("Please select one or more specimen for your samples");
+    experimentalInformationBinder.forField(specimenBox)
+        .asRequired("Please select at least one specimen")
+        .bind(experimentalInformation -> new HashSet<>(experimentalInformation.getSpecimens()),
+            ExperimentalInformation::setSpecimens);
+    MultiSelectComboBox<Analyte> analyteBox = new MultiSelectComboBox<>("Analyte");
+    initComboBoxWithDatasource(analyteBox, List.of(Ontology.BIOASSAY_ONTOLOGY),
+        term -> new Analyte(term.getLabel()));
+    analyteBox.setItemLabelGenerator(Analyte::label);
+    analyteBox.setPlaceholder("Please select one or more analytes for your samples");
+    experimentalInformationBinder.forField(analyteBox)
+        .asRequired("Please select at least one analyte")
+        .bind(experimentalInformation -> new HashSet<>(experimentalInformation.getAnalytes()),
+            ExperimentalInformation::setAnalytes);
+    add(speciesBox, specimenBox, analyteBox);
+  }
+
+  private <T> void initComboBoxWithDatasource(MultiSelectComboBox<T> box, List<Ontology> ontologies,
```

Comment:
please replace with new helper class OntologyFilterConnector.initComboBoxWithOntologyDatasource(...)
(I kept box.addClassNames(CHIP_BADGE, WIDTH_INPUT) here due to the specific references)

Reply:
Thanks for the feedback, it is done 👍